package.json

```diff
@@ -165,25 +161,22 @@
     "react-virtual": "~2.3.2",
     "resolve-url-loader": "^5.0.0",
     "retry": "^0.12.0",
-    "sass-loader": "^10",
+    "sass-loader": "^16.0.3",
     "shpjs": "^4.0.4",
-    "string-replace-webpack-plugin": "^0.1.3",
-    "style-loader": "^0.23.1",
+    "style-loader": "^4.0.0",
     "styled-components": "^5.3.9",
     "svg-sprite-loader": "^6.0.11",
     "terriajs-cesium": "8.0.1",
     "terriajs-cesium-widgets": "5.0.0",
     "terriajs-html2canvas": "1.0.0-alpha.12-terriajs-1",
-    "terriajs-tiff-imagery-provider": "2.13.3",
+    "terriajs-tiff-imagery-provider": "2.13.3-webpack5-3",
     "thredds-catalog-crawler": "0.0.7",
     "ts-essentials": "^5.0.0",
     "typescript": "~5.2.0",
     "urijs": "^1.18.12",
-    "url-loader": "^1.1.2",
-    "webpack": "~4.47.0",
-    "webpack-cli": "^4.10.0",
-    "webpack-dev-server": "^4.15.2",
-    "worker-loader": "^2.0.0"
+    "webpack": "^5.96.1",
```

Comment:
I've seen in the upgrade guide that webpack 5 contains types, but it was listed under internal improvements.

If there are type annotations in the webpack 5 package, I'm guessing the types/webpack dev dependency for terriajs should be removed, and if there aren't I'm guessing the types/webpack version should also be bumped to 5.x?